src/main/java/cuchaz/enigma/gui/util/AbstractListCellRenderer.java

```diff
@@ -0,0 +1,75 @@
+package cuchaz.enigma.gui.util;
+
+import java.awt.Component;
+import java.awt.event.MouseEvent;
+
+import javax.swing.*;
+import javax.swing.border.Border;
+
+public abstract class AbstractListCellRenderer<E> extends JPanel implements ListCellRenderer<E> {
+
+	private static final Border NO_FOCUS_BORDER = BorderFactory.createEmptyBorder(1, 1, 1, 1);
```

Comment:
What is this for? if this is for adding a no focus border, should this be scaled?

Reply:
This is just a copy of the field in DefaultListCellRenderer (A lot of this code is, but with the internal lookup call replaced by look and feel lookup, which seems to do the same thing). There is a scaled border, it gets it from the look and feel in getNoFocusBorder, so it's not necessary here and I want this working as close to DefaultListCellRenderer as possible.